src/main/java/core/basesyntax/Main.java

```diff
@@ -2,6 +2,21 @@
 
 public class Main {
     public static void main(String[] args) {
-        // write your code here
+        Thread myTread = new Thread(() -> {
+            for (int i = 0; i <= 20; i++) {
+                System.out.println(i);
+            }
+        });
+        Thread myDaemonThread = new Thread(new Runnable() {
+            @Override
+            public void run() {
+                while (true) {
+                    System.out.println("I am the daemon thread!");
+                }
+            }
+        });
```

Comment:
I like your solution but you created 2 threads that implement Runnable: 1 using lambda and 1 using anonymous class

let's make one custom thread extend Thread class